lib/config/options/index.ts

```diff
@@ -2357,7 +2357,7 @@ const options: RenovateOptions[] = [
     default: false,
```

Comment:
By the way the config option's name is `keepalive`, but we usually use camelcase. Should we rename the config option like this?

```diff
- name: 'keepalive',
+ name: 'keepAlive',
```

Reply:
`got` appears to use camel case too: https://github.com/search?q=repo%3Asindresorhus%2Fgot%20keepalive&type=code

Reply:
Do you want an issue for renaming to `keepAlive` then?

Reply:
@viceice do you recall why you named it with all lower case?

Reply:
don't remember, maybe I made a typo? need to check where it's passed to got